modules/tensor_mechanics/src/materials/StrainEnergyDensity.C

```diff
@@ -0,0 +1,54 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "StrainEnergyDensity.h"
+#include "RankTwoTensor.h"
+#include "MooseMesh.h"
+
+template <>
+InputParameters
+validParams<StrainEnergyDensity>()
+{
+  InputParameters params = validParams<Material>();
+  params.addClassDescription("Computes the strain energy density");
```

Comment:
I'd expand this to say "This is a combination of the elastic and inelastic components, which is a valid assumption for monotonic behavior"

This should also be discussed in the Markdown for this class, which you'll need to remember to add after the doc freeze.